lib/config/yml_generator_config.dart

```diff
@@ -342,4 +362,32 @@ class YmlGeneratorConfig {
           'Could not generate all models. `$type` is not added to the config file, but is extended. These types are known: ${names.join(',')}');
     }
   }
+
+  ItemType _parseSimpleType(String type) {
+    final listRegex = RegExp(r'List<\s*([^<>]*)\s*>');
+    final mapRegex = RegExp(r'Map<([^<>,]*)\s*,\s*([^<>,]*)\s*>');
```

Comment:
I suggest the following regex: `^\s*[mM][aA][pP]<\s*([a-zA-Z_]*)\s*,\s*([a-zA-Z_]*)\s*>\s*$`
Otherwise:
- `HashMap<Kids,asxc>` will be seen as valid, while it is not
- `  mAP<S,T>  ` will not be seen as valid (ugly, but OK)
- `map< S*c*123, d>` will be seen as valid, whilt it is not

Reply:
Same can be done for List

Reply:
this man's mother language is Regex 🤯

Reply:
I was indeed being a bit too generous with that identifiers should be. Emoji anybody? :D

Reply:
Yet you forgot numbers ;)